api/scanner/scanner_media.go

```diff
@@ -8,24 +8,92 @@ import (
 	"log"
 	"os"
 	"path"
+	"path/filepath"
+	"strings"
 
 	"github.com/pkg/errors"
 	"github.com/viktorstrate/photoview/api/graphql/models"
 )
 
-func scanForSideCarFile(path string) *string {
-	testPath := path + ".xmp"
+func isFileExists(testPath string) bool {
 	_, err := os.Stat(testPath)
 
 	if os.IsNotExist(err) {
-		return nil
+		return false
 	} else if err != nil {
 		// unexpected error logging
 		log.Printf("ERROR: %s", err)
```

Comment:
It is not an actual error. The function is called `isFileExists` so if a file is missing this is not exactly an error - the intention of the check may be to ensure that the file is missing. If you still want to have a debug log, I will add such.

Reply:
I understand. It's more of a warning. I think it would be fine to add some context to the error log, just so if it appears it will be easier to find out where it came frome.